protocol-units/bridge/chains/movement/src/lib.rs

```diff
@@ -34,11 +46,59 @@ pub struct Config {
 	pub rpc_url: Option<String>,
 	pub ws_url: Option<String>,
 	pub chain_id: String,
-	pub signer_private_key: String,
-	pub initiator_contract: MovementAddress,
+	pub signer_private_key: Arc<LocalAccount>,
```

Comment:
This is not thread safe, when it's cloned by more than one resource in the service, it could lead to undefined behaviour.
```suggestion
	pub signer_private_key: Arc<RwLock<LocalAccount>>
```
Wherever its accessed will need to be updated.